src/python_testing/matter_testing_infrastructure/chip/testing/spec_parsing.py

```diff
@@ -823,18 +827,20 @@ def parse_single_device_type(root: ElementTree.Element) -> tuple[dict[int, XmlDe
             classification = next(d.iter('classification'))
             scope = classification.attrib['scope']
             device_class = classification.attrib['class']
+            superset_of_str = classification.attrib.get('superset', None)
```

Comment:
sperset seems to be "what is its superset" not "what superset is this a superset of". 

Explicitly:  `Dimmable Light` has superset being `On/Off Light` which makes sense: all dimmable lights are a on/off light, but not all on/off lights are a dimmable lights.

I think we need to switch naming to remove `of` and then fix comments. Ugly large change, but seems more maintainable for the future.

Reply:
Superset of is the new spec language. The data model was written before this. Superset_of is what we want.